### PR TITLE
VMware - Contribution-  Added command

### PR DESCRIPTION
**Pull request by Bargenish**
## Contributing to Cortex XSOAR Content
Original PR: https://github.com/demisto/content/pull/9933

## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Description
Updated @GavrielFilippov PR:
- Updated README.md file.
- Fixed lint issues
- Removed the unified integration.
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Bargenish** [comment]
@kirbles19 @ShirleyDenkberg this is an existing integration. I converted the README.md to markdown from html.
The new addition is the `change_nic_state` command (yml and readme)

File: Packs/VMware/Integrations/VMware/VMware.yml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+      type: String
+    - contextPath: VMWare.NICState
+      description: VM NIC State (i.e. connected, disconnected, delete)
+      type: String
+  dockerimage: demisto/vmware:1.0-alpine
+  runonce: false
+  script: ''
+  subtype: python2
+  type: python
+fromversion: 6.0.0
```
Comment (bakatzir) [inline Packs/VMware/Integrations/VMware/VMware.yml:252]: i think the old one was from 5.0.0
```suggestion
fromversion: 5.0.0
```

File: Packs/VMware/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "VMware",
   "description": "VMware vCenter server is a centralized management application that lets you manage virtual machines and ESXi hosts centrally.",
   "support": "xsoar",
-  "currentVersion": "1.0.1",
+  "currentVersion": "1.0.2",
```
Comment (bakatzir) [inline Packs/VMware/pack_metadata.json:5]: you can do a 1.1.0, your call

File: Packs/VMware/Integrations/VMware/VMware.py [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+import ssl
+import sys
+from datetime import datetime
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from cStringIO import StringIO
+from pyVim.connect import Disconnect, SmartConnect
+from pyVim.task import WaitForTask
+from pyVmomi import vim, vmodl
```
Comment (bakatzir) [inline Packs/VMware/Integrations/VMware/VMware.py:13]: are all these imports sorted and needed? if not, please tend to it.

Comment (Bargenish) [reply Packs/VMware/Integrations/VMware/VMware.py:13]: All are needed (I removed the unnecessary ones).
What's our best practice regarding sorting imports?  This is how it sorted on the original integration.

Comment (bakatzir) [reply Packs/VMware/Integrations/VMware/VMware.py]: i like using: https://pypi.org/project/isort/
isort {path_to_py_file}

**Review by bakatzir — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@bakatzir Doc review completed.

**Pull request merged**